src/utils/githubApi.ts

```diff
@@ -87,6 +87,8 @@ export function getGithubApiToken(): string {
   return githubApiToken;
 }
 
+const _GitHubClientCache: Record<string, Octokit> = {};
```

Comment:
Maybe use `Map` instead? It has a nicer API to work with caches (`has` vs. coercing with `!`)

Reply:
Since we are using strings as keys, using a good-old-object is the better way forward. I agree with `Map` API but typing and perf around it is still an issue to this day :(